.github/workflows/update-translations.yml

```
@@ -8,12 +11,30 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
       - name: Fetch and merge from Weblate
         # The commit message is generated in Weblate; see https://hosted.weblate.org/addon/17163/
         run: |
           git remote add weblate https://hosted.weblate.org/git/zulip/zulip-flutter/
           git fetch weblate
           git merge --ff-only weblate/main
+
+      - name: Clone Flutter SDK
+        # We can't do a depth-1 clone, because we need the most recent tag
+        # so that Flutter knows its version and sees the constraint in our
+        # pubspec is satisfied.  It's uncommon for flutter/flutter to go
+        # more than 100 commits between tags.  Fetch 1000 for good measure.
+        run: |
+          git clone --depth=1000 -b main https://github.com/flutter/flutter ~/flutter
+          TZ=UTC git --git-dir ~/flutter/.git log -1 --format='%h | %ci | %s' --date=iso8601-local
+          echo ~/flutter/bin >> "$GITHUB_PATH"
+
+      - name: Update generated code
+        run: |
+          ./tools/check l10n --fix
```

nit:
```suggestion
          tools/check l10n --fix
```

The rule is that once there's a slash in the command name, it means a file by that name, and there's no search in PATH (or for shell functions or builtins). Docs:
https://www.gnu.org/software/bash/manual/bash.html#Command-Search-and-Execution
(and POSIX says the same thing but is more annoying to reference:
https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/V3_chap02.html#tag_18_09_01 )

So a leading `./` is useful only as a trick to get a slash in there when the file's name would otherwise have no slash.

---

I'm aware that it's not _required_ -- however, I like its specificity, and that it makes the behaviour consistent and correct if someone naïvely moves the file up a directory.

Regardless, I'll defer to your preference here.